scalatest/src/main/scala/org/scalatest/tools/ScalaTestAntTask.scala

```diff
@@ -995,7 +995,7 @@ class ScalaTestAntTask extends Task {
   // Class to hold data from <suite> elements.
   //
   private class SuiteElement {
-    private var className: String = null
+    private var className: String | Null = null
```

Comment:
**Nullable variable**
Variables that really can be null and should therefore have a `| Null` type.